### PR TITLE
run: fixes upon new commits

### LINKED ISSUE #210: Make run greate again
Need to change the behavior of the tt run command:

* tt run should only run the specified scripts. E.g.:
```sh
$ tt run foo.lua
$ tt run /tmp/foo.lua
```
* tt run should not start instances.
* tt run must pass arguments to the script. E.g.:
```sh
$ tt run foo.lua -- a b c
```

* tt run must read and use only `data_dir` from tarantol.yaml

### PR DESCRIPTION
**Pull request by better0fdead**
Removed support of multi instance apps.
Removed start from instances enabled dir, only valid option is to specify script directly.
Updated tests for new behaviour, added new tests.
Removed 'Starting an instance...' log message since it is misleading.
Closes #210

### CODE REVIEW

File: test/integration/run/test_run.py [hunk 1]
```diff
@@ -36,25 +32,65 @@ def test_running_flag_version(tt_cmd, tmpdir):
...
+    run_output = instance_process.stdout.readline()
+    assert re.search(r"123", run_output)
+
+
+def test_running_arg(tt_cmd, tmpdir):
     # Copy the test application to the "run" directory.
+    test_app_path = os.path.join(os.path.dirname(__file__), "test_app", "test_app_arg.lua")
+    shutil.copy(test_app_path, tmpdir)
 
     # Start an instance.
```
Comment (0x501D) [inline test/integration/run/test_run.py:58]: This comment is outdated. We are not starting instances with `run` .

File: test/integration/run/test_run.py [hunk 2]
```diff
@@ -36,25 +32,65 @@ def test_running_flag_version(tt_cmd, tmpdir):
...
         text=True
     )
-    start_output = instance_process.stdout.readline()
-    assert re.search(r"   • Running an instance...   \n", start_output)
     run_output = instance_process.stdout.readline()
     assert re.search(r"123", run_output)
+
+
+def test_running_missing_script(tt_cmd, tmpdir):
+    # Start an instance.
```
Comment (0x501D) [inline test/integration/run/test_run.py:72]: Same as above.

File: test/integration/run/test_run.py (hunk 1 above)
Comment (better0fdead) [reply test/integration/run/test_run.py]: Changed

File: test/integration/run/test_run.py (hunk 2 above)
Comment (better0fdead) [reply test/integration/run/test_run.py]: Changed

**Review by 0x501D — APPROVED** [review]
LGTM

**Review by LeonidVas — CHANGES_REQUESTED** [review]
During testing, I found several bugs:
* after exiting from tarantool (Tarantool 2.10.3-0-g8aca8a19a) the terminal looks broken
image
* when SIGINT is sent to a process, the process is terminated, but this is incorrect behavior for tarantool

Pavel said he would investigate the problems contact him.

**Pull request merged**